src/openrct2-ui/windows/Scenery.cpp

```diff
@@ -630,10 +630,7 @@ static Widget WindowSceneryBaseWidgets[] = {
             if (gWindowSceneryScatterEnabled)
                 pressed_widgets |= (1uLL << WIDX_SCENERY_BUILD_CLUSTER_BUTTON);
 
-            widgets[WIDX_SCENERY_ROTATE_OBJECTS_BUTTON].type = WindowWidgetType::Empty;
-            widgets[WIDX_SCENERY_EYEDROPPER_BUTTON].type = WindowWidgetType::Empty;
             widgets[WIDX_SCENERY_BUILD_CLUSTER_BUTTON].type = WindowWidgetType::Empty;
-            widgets[WIDX_RESTRICT_SCENERY].type = WindowWidgetType::Empty;
```

Comment:
This is not correct. _Only_ the line that removes the `WIDX_SCENERY_EYEDROPPER_BUTTON` should be dropped here.